src/data/data.json

```diff
@@ -0,0 +1,25 @@
+{
+  "automobile": [

```

Comment:
just to know, this is the key to be accessed when you're using map;

Take a look how map works:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map